src/lib/jdk/KeyTool.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import * as fs from 'fs';
+import * as util from '../util';
+import {JdkHelper} from './JdkHelper';
+
+export interface CreateKeyOptions {
+  path: string;
+  alias: string;
+  cn: string; // First and Last Name
```

Comment:
Is there any reason other than brevity for these variable names? If not, can we just use `fullName`, `orgUnit`, `org` and `country`?

Reply:
Those just match the parameters passed to keytool. I agree  it's better to have more descriptive names.